CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
-  Thanks to [Debs](https://github.com/debsdee) and [Joe](https://github.com/joelanman) for raising this issue.
+If you're passing a class name, put the mixin inside your selector. For example:
 
-  ([PR #1353](https://github.com/alphagov/govuk-frontend/pull/1353))
+```scss
+.your-class-name {
+  @include govuk-grid-column(...)
+}
+```
 
-- Fix HTML elements in tabs label breaking
+You can no longer use the `govuk-grid-row` mixin. You can replace it with the `.govuk-grid-row` class in your HTML.
 
-  ([PR #1351](https://github.com/alphagov/govuk-frontend/pull/1351))
+You must also replace calls to the `grid-width` mixin with calls to the `govuk-grid-width` mixin.
 
-- Fix tabs keyboard navigation bug in IE8
+Pull requests:
 
-  Users were unable to tab between tab panels using the keyboard and had to
-  use their mouse to toggle between panels.
+- [#1376: Remove $class param from @govuk-grid-column mixin](https://github.com/alphagov/govuk-frontend/pull/1376)
+- [#1342: Remove grid-width mixin](https://github.com/alphagov/govuk-frontend/pull/1342)
+- [#1343: Removes govuk-grid-row mixin](https://github.com/alphagov/govuk-frontend/pull/1343)
 
-  ([PR #1359](https://github.com/alphagov/govuk-frontend/pull/1359))
+### Rename the border width variable
 
-- Update accordion focus styles to remove firefox outlines
+If you use Sass and you’ve extended or created components that use the border width variable, rename `$govuk-border-width-mobile` to `$govuk-border-width-narrow`.
 
-  ([PR #1324](https://github.com/alphagov/govuk-frontend/pull/1324))
+[Pull request #1287: Rename border-width-mobile to reflect how it's used](https://github.com/alphagov/govuk-frontend/pull/1287)
 
-- Rename `$govuk-border-width-mobile` to `$govuk-border-width-narrow`
+## New features
 
-  This better reflects how the variable is used.
+### Add attributes to table headings
 
-  Also make the error summary border the standard width on mobile.
+You can now add attributes like classes, rowspan and colspan to table row headers.
 
-  ([PR #1287](https://github.com/alphagov/govuk-frontend/pull/1287))
+[Pull request #1367: Allow for classes, rowspan, colspan and attributes on row headers](https://github.com/alphagov/govuk-frontend/pull/1367). Thanks to [edwardhorsford](https://github.com/edwardhorsford).
 
-- Fixes govuk-template focussed link colour overriding govuk-frontends
+### Use page wrappers without a modifier
 
-  govuk-template uses specific css selector`a:link:focus` whereas
-  govuk-frontend uses `.govuk-link:focus`. This fix makes govuk-frontend
-  selector more specific `.govuk-link:link:focus` but only when compatibility
-  mode is being used.
+If your page wrapper does not have components above the main section, you no longer need to use the [`govuk-main-wrapper--l` modifier class](https://design-system.service.gov.uk/styles/layout/#exploded-view-of-page-wrappers-without-a-back-link-breadcrumbs-or-phase-banner) to increase the vertical space above the content.
```

Comment:
This needs to be updated based on https://github.com/alphagov/govuk-frontend/pull/1493